### DISCUSSION TITLE
Gas limit

### DISCUSSION THREAD

**Discussion by Osuolale1** [Cyfrin/foundry-full-course-cu#443] [General]
![IMG-20230725-WA0010](https://github.com/Cyfrin/foundry-full-course-f23/assets/101529913/01f19f82-bfd4-4fbc-9bde-8fbc404a1ba3)
Having encounter with the gas limit

**Comment by OMGitsNIK** [comment]
I would suggest you to send the markdown of your code instead of a screen shot, as it will make it easier for us to implement your code and see what's wrong. 

To use markdown, you simply need to use ``` before and after your code and also mention the language of the code.
eg)
```solidity
//SPDX-License-Identifier: MIT
pragma solidity 0.8.18
contract apple {
     function mango( ) public { }
}
```
For more details on markdown you can visit [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
